src/queries/token/usePointsBreakdownQuery.ts

```diff
@@ -0,0 +1,124 @@
+
+    vaultsPoints: '-',
+    lpPoints: '-',
+    tradingPoints: '-',
+
+    thalesStaked: '-',
+    thalesDivider: '-',
+    totalPoints: '-',
+};
+
+const usePointsBreakdownQuery = (
+    walletAddress: string,
+    networkId: Network,
+    options?: UseQueryOptions<PointsData | undefined>
+) => {
+    return useQuery<PointsData | undefined>(
+        QUERY_KEYS.Token.PointsBreakdown(walletAddress, networkId),
+        async () => {
+            const { stakingThalesContract } = snxJSConnector;
+            const { stakingBonusRewardsManager } = snxJSConnector;
+
+            if (!walletAddress) {
+                return DEFAULT_POINTS_BREAKDOWN_DATA;
+            }
+
+            try {
+                const period = await stakingThalesContract?.periodsOfStaking();
+                const [
+                    vaultsPoints,
+                    lpPoints,
+                    tradingPoints,
+                    vaultsMultiplier,
+                    lpMultiplier,
+                    tradingMultiplier,
+                    stakingMultiplier,
+                    stakedBalance,
+                    thalesDivider,
+                ] = await Promise.all([
+                    stakingBonusRewardsManager?.getEstimatedCurrentVaultPoints(walletAddress),
+                    stakingBonusRewardsManager?.getEstimatedCurrentLPsPoints(walletAddress),
+                    stakingBonusRewardsManager?.userTradingBasePointsPerRound(walletAddress, period),
+                    stakingBonusRewardsManager?.vaultsMultiplier(),
+                    stakingBonusRewardsManager?.lpMultiplier(),
+                    stakingBonusRewardsManager?.tradingMultiplier(),
+                    stakingBonusRewardsManager?.getStakingMultiplier(walletAddress),
+                    stakingThalesContract?.stakedBalanceOf(walletAddress),
+                    stakingBonusRewardsManager?.stakingBaseDivider(),
+                ]);
+
+                return {
+                    vaultsVolume: formatCurrencyWithKey(
+                        USD_SIGN,
+                        bigNumberFormatter(vaultsPoints) / bigNumberFormatter(vaultsMultiplier)
+                    ),
+                    lpVolume: formatCurrencyWithKey(
+                        USD_SIGN,
+                        bigNumberFormatter(lpPoints) / bigNumberFormatter(lpMultiplier)
+                    ),
+                    tradingVolume: formatCurrencyWithKey(
+                        USD_SIGN,
+                        bigNumberFormatter(tradingPoints) / bigNumberFormatter(tradingMultiplier)
+                    ),
+                    vaultsMultiplier: bigNumberFormatter(vaultsMultiplier),
+                    lpMultiplier: bigNumberFormatter(lpMultiplier),
+                    tradingMultiplier: bigNumberFormatter(tradingMultiplier),
+                    stakingMultiplier: formatCurrency(bigNumberFormatter(stakingMultiplier) + 1),
+                    vaultsPoints: formatCurrency(bigNumberFormatter(vaultsPoints)),
```

Comment:
Formating of the currency should be done outside the query when we want to display data.